doc/source/admin/interactive_environments.rst

```diff
@@ -176,13 +199,14 @@ proxying to GIE and other visualization plugin static content.
 Docker on Another Host
 ^^^^^^^^^^^^^^^^^^^^^^
 
-There are many reasons to run Interactive Environments on a separate host and
-not on your webserver, serving Galaxy. This feature has been available since
+You might want to run your IEs on a host different to the one that hosts your
+Galaxy webserver, since IEs on the same host as the webserver compete for
+resources with that webserver. This feature has been available since
```

Comment:
"with that webserver and introduce some security considerations which could be mitigated by moving containers to a separate host"